application/server.py

```diff
@@ -66,6 +66,13 @@ def handle_create_room_form():
         confirmed_password = request.form.get('confirmed-room-password')
 
         if room_password == confirmed_password:
+<<<<<<< HEAD
```

Comment:
Ummm. I think this is the result of a merge conflict. You'll need to solve this.

Reply:
i believe i fixed it now